### DISCUSSION TITLE
Aspire vs Radius

### DISCUSSION THREAD

**Discussion by thangchung** [microsoft/aspire#1093] [Q&A] [answered]
I had a look at [Radius](https://radapp.io/) a couple of weeks ago, and really exiting about how it can help to solve the problem of development and deployment into cloud-native platforms such as Kubernetes (like platform orchestrator in My POV).

And now with .NET Conf 2023, I have seen Aspire, to be honest; that's really cool as well (platform orchestration support).

My question is any chance we can combine the power of both? After seeing some of the implementations and samples, it might really hard to use both of them. Could anyone in the MS development team shed light on it? I appreciated that.

**Comment by davidfowl** [comment, marked as answer]
This was answered here https://github.com/dotnet/aspire/discussions/811

**Reply by davidfowl** [reply]
Radis team just merged their first PR https://github.com/microsoft/aspire/pull/18696

**Comment by DamianEdwards** [comment]
Closing this out as it appears answered.
